app/javascript/packages/document-capture/components/in-person-location-step.tsx

```diff
@@ -30,9 +30,22 @@ interface FormattedLocation {
   weekdayHours: string;
 }
 
+const getCsrfToken = () => {
+  const meta: HTMLMetaElement | null = document.querySelector('meta[name="csrf-token"]');
+
+  return meta?.content;
+};
+
 export const LOCATIONS_URL = '/verify/in_person/usps_locations';
+const getUspsLocations = () => {
```

Comment:
Not a blocker, but there's an opportunity here to standardize common API request behavior with a new package, like...

```tsx
import { request } from '@18f/identity-request';

const getUspsLocations = () => request.post(LOCATIONS_URL);
```

Reply:
What if we added a `fetchWithCSRF` that just adds in the CSRF header to a `fetch` call?

Reply:
We'll need the JSON handling as well. I recall we had some [back-and-forth on this in a prior incarnation](https://github.com/18F/identity-idp/pull/6305#discussion_r869446680) of this helper. I'm sorta of the mind that it should "just work", all batteries / magic included, since I think almost all (or all) instances we'd be using it, we'd want this behavior?

Reply:
ahaha I knew this was familiar, thanks for digging that up. I feel the same, but generally agree that yes, some helper functions to help with JSON and with CSRS would be great